platform-sdk/swirlds-merkle/src/test/java/com/swirlds/virtual/merkle/reconnect/VirtualMapMerkleDbReconnectTest.java

```diff
@@ -57,37 +55,24 @@
 @DisplayName("Virtual Map MerkleDB Reconnect Test")
 class VirtualMapMerkleDbReconnectTest extends VirtualMapMerkleDbReconnectTestBase {
 
-    private static MerkleDbSettings originalSettings;
+    private static Configuration originalConfig;
 
     @BeforeAll
-    static void beforeAll() throws Exception {
-        originalSettings = MerkleDbSettingsFactory.get();
-        MerkleDbSettingsFactory.configure(new TestMerkleDbSettings(originalSettings) {
-            @Override
-            public int getKeySetBloomFilterHashCount() {
-                return 10;
-            }
-
-            @Override
-            public long getKeySetBloomFilterSizeInBytes() {
-                return 2 * MEBIBYTES_TO_BYTES * BYTES_TO_BITS;
-            }
+    static void beforeAll() {
+        originalConfig = ConfigurationHolder.getInstance().get();
 
-            @Override
-            public long getKeySetHalfDiskHashMapSize() {
-                return 10_000;
-            }
+        final Configuration config = new TestConfigBuilder()
+                .withValue("merkleDb.keySetBloomFilterSizeInBytes", "16777216")
+                .withValue("merkleDb.keySetHalfDiskHashMapSize", "10000")
+                .withValue("merkleDb.keySetHalfDiskHashMapBuffer", "1000")
+                .getOrCreateConfig();
 
-            @Override
-            public int getKeySetHalfDiskHashMapBuffer() {
-                return 1_000;
-            }
-        });
+        ConfigurationHolder.getInstance().setConfiguration(config);
     }
 
     @AfterAll
     static void afterAll() {
-        MerkleDbSettingsFactory.configure(originalSettings);
+        ConfigurationHolder.getInstance().setConfiguration(originalConfig);
```

Comment:
instead of holding the originalConfig you could just do `setConfiguration(new TestConfigBuilder().getOrCreateConfig())`. But in general I would say that we should remove the afterAll anyway since that was needed in the past to reset the static config between test classes. I assume that we finally have an initialization in each tests where the config is important. So maybe you just test if everything is working after you removed the afterAll